### PR TITLE
Style Fixes

### PR DESCRIPTION
**Pull request by Bowserinator**
Some CSS style fixes

1. Login button is now in the navbar and on the right.
2. Back button is now absolutely positioned and no longer interferes with the height placement of the Paper
3. The Paper now has increased padding
4. The repo settings now has smaller and more compact text-input placement. No more giant text input areas!
5. Text input labels in the repo settings can actually be read when focused (Before they were too dark to see)

Dashboard style

### CODE REVIEW

**Comment by handicraftsman** [comment]
(btw i actually checked the code)

**Comment by handicraftsman** [comment]
But what are these build fails?

**Comment by wolfy1339** [comment]
Totally unrelated. It's jest being dumb. Blame `create-react-app`
________________________________

File: dashboard/src/GitHub.js [hunk 1]
```diff
@@ -11,9 +11,9 @@ import Button from '@material-ui/core/Button';
 */
 function GitHub(props) {
   return (
-    <a href="/login">
-      <Button variant="contained" color="black" {...props}>
-        <SvgIcon viewBox="0 0 42.607 41.555" titleAccess="GitHub logo">
+    <a href="/login" className="login-button">
+      <Button className="login-button-btn" variant="contained" color="primary" {...props}>
+        <SvgIcon viewBox="0 0 42.607 41.555" titleAccess="GitHub logo" viewBox="0 0 24 24" className="login-button-svg">
```
Comment (wolfy1339) [inline dashboard/src/GitHub.js]: Why 2 viewboxes?

Comment (handicraftsman) [reply dashboard/src/GitHub.js]: Btw why SvgIcon and not a material icon? There're no material alternatives?

**Comment by wolfy1339** [comment]
Unfortunately not. It's the GitHub logo, and according to mui-org/material-ui#8006, they won't add any other icons then the material icons
________________________________

**Comment by wolfy1339** [comment]
@BWBellairs you like?

**Pull request merged**